pennylane/qaoa/cost.py

```diff
@@ -244,3 +245,217 @@ def maxcut(graph):
     )
     H = edge_driver(graph, ["10", "01"]) + identity_h
     return (H, qaoa.x_mixer(graph.nodes))
+
+
+def max_independent_set(graph, constrained=True):
+    r"""Returns the QAOA cost Hamiltonian and the recommended mixer corresponding to the Maximum Independent Set problem,
+    for a given graph.
```

Comment:
```suggestion
    r"""For a given graph, returns the QAOA cost Hamiltonian and the recommended mixer corresponding to the Maximum Independent Set problem.
```
I feel like I made this comment before :thinking: